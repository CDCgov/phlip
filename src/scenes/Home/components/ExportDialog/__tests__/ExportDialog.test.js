import React from 'react'
import { shallow } from 'enzyme'
import { ExportDialog } from '../index'

const props = {
  onClose: jest.fn(),
  onChooseExport: jest.fn(),
  open: true,
  projectToExport: {
    id: 1,
    text: '',
    user: { id: null, firstName: '', lastName: '' },
    exportType: null
  },
  inProgress: false
}

describe('Home - Export Dialog', () => {
  test('should render correctly', () => {
    expect(shallow(<ExportDialog {...props} />)).toMatchSnapshot()
  })
})
