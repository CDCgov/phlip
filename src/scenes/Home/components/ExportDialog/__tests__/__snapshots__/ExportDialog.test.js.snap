// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home - Export Dialog should render correctly 1`] = `
<WithStyles(Modal)
  id="export-modal"
  onClose={[Function]}
  open={true}
>
  <ModalTitle
    title="Choose export type"
  />
  <WithStyles(Divider) />
  <ModalContent
    style={
      Object {
        "minWidth": 400,
        "padding": 0,
      }
    }
  >
    <WithStyles(List)>
      <ListItem
        disabled={false}
        inProgress={false}
        isSubItem={false}
        item={
          Object {
            "displayText": "Codebook",
          }
        }
        onExport={[Function]}
        showAvatar={false}
      />
      <ListSection
        expanded={false}
        inProgress={
          Object {
            "type": null,
            "user": null,
          }
        }
        onExpand={[Function]}
        onExport={[Function]}
        section="numeric"
        sectionText="Coded Data - Numeric"
      />
      <ListSection
        expanded={false}
        inProgress={
          Object {
            "type": null,
            "user": null,
          }
        }
        onExpand={[Function]}
        onExport={[Function]}
        section="text"
        sectionText="Coded Data - Text"
      />
    </WithStyles(List)>
  </ModalContent>
  <WithStyles(ModalActions)
    actions={
      Array [
        Object {
          "onClick": [Function],
          "otherProps": Object {
            "aria-label": "Close modal",
          },
          "type": "button",
          "value": "Close",
        },
      ]
    }
  />
</WithStyles(Modal)>
`;
